### PR TITLE
Added a filter to /devicesview to not return devices that have no ima…

### PR DESCRIPTION
**Pull request by AaronH88**
…ge-id.

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [X] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `go fmt ./...` to check that my code is properly formatted
- [X] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

File: pkg/services/devices.go [hunk 1]
```diff
@@ -585,7 +585,7 @@ func (s *DeviceService) GetDevicesView(limit int, offset int, tx *gorm.DB) (*mod
 	}
 
 	var storedDevices []models.Device
-	if res := tx.Limit(limit).Offset(offset).Where("account = ?", account).Find(&storedDevices); res.Error != nil {
+	if res := tx.Limit(limit).Offset(offset).Where("account = ?", account).Not("image_id = NULL").Find(&storedDevices); res.Error != nil {
```
Comment (ldjebran) [inline pkg/services/devices.go:588]: may be 
```go
tx.Limit(limit).Offset(offset).Where("account = ? AND image_id IS NOT NULL", account).Find(&storedDevices)
```

what do you think ?
or   .Not("image_id = NULL")  will be converted to image_id IS NOT NULL   ?

Comment (AaronH88) [reply pkg/services/devices.go:588]: Sure! Ill change it :smile:

**Review by ldjebran — APPROVED** [review]
LGMT

**Pull request merged**